docs/UserGuide.adoc

```diff
@@ -209,79 +172,104 @@ Format: `find n\[name]`
 
 Example:
 
-* `find n\Jane doe` +
-Finds a saved contact named Jane Doe.
+* Suppose you want to find Jane doe within NOVA,
++
+`find n\Jane doe`
++
+NOVA will find and list saved contacts named Jane Doe.
+
+* Suppose you want to find Jane within NOVA,
++
+`find n\Jane`
++
+NOVA will find and list saved contacts named Jane.
 
-* `find n\Jane` +
-Finds all saved contacts named Jane.
+==== Edit Contact: `edit`
+You can edit the contacts that you have added. If the contact you want to edit does not exist, NOVA will let you know.
 
-==== Add Picture to Contact: `addpic`
-You can add a profile picture to a contact in your address book.
-If a profile picture had already been added, any newly added profile picture will be replaced
+At least one of the optional fields must be provided. Optional fields are `n\[name]`, `p\[phone number]`,
+`e\[email address]` or `c\[classmate/teammate]`.
 
-Format: `addpic i\[index] pa\[path]`
+Format: `edit i\[index] n\[name] p\[phone number] e\[email address] c\[classmate/teammate]`
 
 [NOTE]
 ====
-* Adds profile picture of the person at the specified `[index]` in the displayed person list
+* Edits the contact at the specified `[index]` in the displayed contact list
 * The `[index]` must be a positive integer. (E.g. 1, 2, 3, ...)
-* Absolute file path is used for `[path]`
+* `[name]` and `[classmate/teammate]` are not case-sensitive. (E.g. "Jane doe" is the same as "jane Doe")
+* The name you add will be automatically formatted. (E.g. "jane doe" will become "Jane Doe")
+* You can only edit either `classmate` or `teammate` as category
 ====
 
 Example: +
-`addpic i\2 pa\C:\Users\aloha\Desktop\aloha.png`
+Suppose you want to edit the first contact's phone number in your address book after using `list`,
+`list c\[classmate/teammate]` or `find` command,
 
-Sets the profile picture of the second contact in your address book to aloha.png.
+`edit i\1 p\88888888`
 
-==== Delete Picture in Contact: `deletepic`
-You can delete the profile picture of a contact in your address book.
+NOVA will edit the phone number of the first contact in your address book to 88888888.
```

Comment:
If I understand correctly, you would be modifying the first contact of the list and not the address book.